Create beta-deprecation-disclaimer.md (#9630)
microsoftgraph/microsoft-graph-docs-contrib: api-reference/includes/beta-deprecation-disclaimer.md

---
author: archanayerragangu
ms.topic: include
ms.date: 08/20/2025
ms.localizationpriority: medium
---

<!-- markdownlint-disable MD041-->

> > [!DEPRECATION NOTICE]

Reviewer: @ArchanaYerragangu, before I can review this PR. Please review these guidelines. Can you confirm that you've already followed this process? (https://eng.ms/docs/products/microsoft-graph-service/microsoft-graph/planning/api-deprecation-process). Also, please look at the doc updates needed when deprecating elements: (https://eng.ms/docs/products/microsoft-graph-service/microsoft-graph/document-apis/reference-content/document-deprecated-apis).

> > [!DEPRECATION NOTICE]

Reviewer: ```suggestion
> [!Note]
```

> > The Microsoft Graph beta APIs related to device management under the `teamworkDevice` resource type will be deprecated by November 2025. These APIs will no longer be supported after that time.

Reviewer: ```suggestion
> The Microsoft Graph beta APIs related to device management under the `teamworkDevice` resource type will be deprecated by November 2025 and will no longer be supported after that date.
```
